### PR TITLE
Remove SQL-FS

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Signed-off-by: Jens Wiklander <jens.wiklander@linaro.org>

Depends on https://github.com/OP-TEE/optee_os/pull/1490

### CODE REVIEW

File: tee-supplicant/src/optee_msg_supplicant.h [hunk 1]
```diff
@@ -48,14 +48,8 @@
...
- * SQLite file system access
- */
-#define OPTEE_MSG_RPC_CMD_SQL_FS	8
-
-/*
- * Define protocol for messages with .cmd == OPTEE_MSG_RPC_CMD_FS or
- * .cmd == OPTEE_MSG_RPC_CMD_SQL_FS and first parameter has the attribute
- * OPTEE_MSG_ATTR_TYPE_VALUE_INPUT.
+ * Define protocol for messages with .cmd == OPTEE_MSG_RPC_CMD_FS and first
+ * parameter has the attribute OPTEE_MSG_ATTR_TYPE_VALUE_INPUT.
```
Comment (etienne-lms) [inline tee-supplicant/src/optee_msg_supplicant.h]: `OPTEE_MSG_RPC_CMD_SQL_FS_RESERVED` as optee_os conterpart.

File: tee-supplicant/src/tee_fs.h [hunk 2]
```diff
@@ -69,8 +69,6 @@ struct tee_fs_rpc {
 #define TEE_FS_RMDIR     13
 #define TEE_FS_ACCESS    14
 #define TEE_FS_LINK      15
```
Comment (etienne-lms) [inline tee-supplicant/src/tee_fs.h]: I would also keep those. Not mess API.

**Comment by jforissier** [comment]
I'm also in favor of keeping the values as "reserved".
In any case:
`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by jenswi-linaro** [comment]
Comments addressed, rebased, tag applied.

**Pull request merged**